packages/react/src/date-picker/reactDayPickerStyles.ts

```diff
@@ -36,27 +49,18 @@ export const reactDayPickerStyles = {
 		...fontGrid('lg', 'nospace'),
 	},
 	// Left / right arrows
-	'.rdp-nav_button': {
-		position: 'absolute',
-		top: 0,
-		display: 'flex',
-		alignItems: 'center',
-		justifyContent: 'center',
-		width: '2rem',
-		height: '2rem',
-		borderRadius: tokens.borderRadius,
-		color: boxPalette.foregroundAction,
-		'&:hover': { color: boxPalette.foregroundText },
-		...focusStyles,
-	},
-	'.rdp-nav_button_previous': {
+	'.rdp-button_previous': {
```

Comment:
Can you just do this? ```suggestion
	'.rdp-button_previous, .rdp-button_next': {
```